app/marketing/mails.py

```diff
@@ -1997,3 +2011,47 @@ def remember_your_cart(profile, cart_query, grants, hours):
             send_mail(from_email, to_email, subject, text, html, categories=['marketing', func_name()])
     finally:
         translation.activate(cur_language)
+
+def tribe_hackathon_prizes(hackathon):
+    from dashboard.models import TribeMember, Sponsor
+    from marketing.utils import generate_hackathon_email_intro
+
+    sponsors = hackathon.sponsors.all()
```

Comment:
we have a problem here, should use sponsor_profiles we not longer use Sponsors to control hackathons

Reply:
@octavioamu @PixelantDesign I have a PR up at #7589